make.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+elm-make dapp/client/App.elm --output=dapp/client/app.js
```

Comment:
I'm not sure how you're calling `elm-make` but this was helpful for me to stop having to think about it every time. There are better ways to structure things, but not worrying about that yet.